test/firtool/spec/refs/force_nonpassive.fir

```diff
@@ -1,5 +1,6 @@
 ; RUN: firtool %s -verify-diagnostics
 ; RUN: firtool %s -preserve-aggregate=all -scalarize-top-module=false -verify-diagnostics
+; Spec says this is valid, but is now actively rejected.
```

Comment:
What's the plan to reconcile the spec with firtool here?

Reply:
Easy part: Drop the section promising this specifically works (hence the example), and adding language clarifying.

That said, it's not great because our requirement to split apart bundles with flips is an implementation detail.

Will think some more, this seemed easy because our pipeline really fights this and would rather reject it neatly than kinda-sometimes support it (see the first example in the recent MergeConnections issue).